pkg/vpa/vpa.go

```diff
@@ -22,9 +22,9 @@ import (
 	"strconv"
 	"strings"
 
+	"github.com/samber/lo"
```

Comment:
this is unnecessary code churn, and no longer in alphabetical order, can we please revert this?